### ISSUE TITLE
Cloudflare cookie for anti-DDoS check always empty (2ch)

### ISSUE THREAD

**Issue by ViktorVoloshko** [K1rakishou/Kuroba-Experimental#786]
Title said enough I think. Actually I have seen cookie code once there but after I tried to post it disappeared and I wasn't able to get a new one. Video:
https://user-images.githubusercontent.com/87974065/184239627-d17f983c-3ba9-45fc-8208-114bf3c62ab7.mp4

**Comment by K1rakishou** [comment]
Fixed in https://github.com/K1rakishou/Kuroba-Experimental/commit/039ee415712371f9eb2d025066fb6b48c93aacd5

**Comment by ViktorVoloshko** [comment]
Also you need to disable OkHttp via HTTP/2 to get rid of this nasty checks. I had it enabled because images were fine on .life and there was no need in disabling. Now everything works fine, thx.

**Comment by K1rakishou** [comment]
>Now everything works fine, thx.

I haven't released the fix yet, though. The actual problem was in a race condition because of which the cookie would get removed right after being set. I don't think those settings have anything to do with this.

**Comment by ViktorVoloshko** [comment]
Well, somehow I am able to use .life without this fix and turned off that experimental feature. But when I seen Cloudflare check last time it said 2ch.hk and I thought it's just a bug with site. Now I think that I'm using .hk even with .life selected.

**Comment by K1rakishou** [comment]
Nah, it's probably just so happened that in this one time the race condition didn't happen so the cookie was properly set so now it should be fine until the cookie's lifetime is over. 

>But when I seen Cloudflare check last time it said 2ch.hk 

Yes, I hardcoded it to always open 2ch.hk/challenge because earlier when they first introduced the anti-ddos checks (like a year ago?) that was the only url that could trigger the checks. Now it seems they work on almost any url, so I changed that as well.

Anyway, I improved Cloudflare detection so now the Cloudflare screen should show up as soon as the very first request gets rejected by it, in other words you won't have to open up the captcha window to pass Cloudflare checks anymore.
